src/core/reducers/addons.js

```diff
@@ -286,10 +286,55 @@ export function createInternalAddon(
   return addon;
 }
 
-const initialState = {};
+type AddonID = string;
```

Comment:
maybe I could switch to `number` actually

Reply:
mmh no, it expects a `string` in:

```
Object.keys(addons).map((key: AddonID) => addons[key]);
```

We should do this because `Object.values()` cannot be used with Flow...